azure-pipelines.yml

```
@@ -148,23 +148,23 @@ stages:
   # Like template `eng/common/templates/jobs/source-build.yml`
   - job: Source_Build_Managed
     displayName: Source-Build (Managed)
-    container: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream8
+    container: mcr.microsoft.com/dotnet-buildtools/prereqs:cbl-mariner-2.0
```

Source build legs should continue to use centos as we don't ship a SB product via mariner.

```suggestion
      container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream9'
```